### ISSUE TITLE
Update to aiohttp>=1.2.0

### ISSUE THREAD

**Issue by Kern3l** [klen/muffin#55]
Hey,

when using `aiohttp.ClientSession` and POSTing to a certain page **I get a bug that is fixed in _aiohttp==1.2.0_**. I'm forced to use _requests_ for this with self-added async support. I'd like to use `aiohttp.ClientSession` for this like I do in all other cases. This requires _aiohttp>=1.2.0_, I've tested. **Is there any chance for an upgrade?**

There are breaking changes in _aiohttp>1.0.5_. I started upgrading _muffin_ to support 1.2.0, but I was discouraged by also needing to upgrade plugins (`sentry` and `redis`) - the time required was too big to commit to it at work.

In short, changes to be made in _muffin_ (the list may be incomplete):
- `RequestHandlerFactory` became `web.Server` and has one parameter less
- some classes changed places
- `from aiohttp include *` in \_\_init__.py possibly has to go away (or be updated)
- `StaticRoute` doesn't exist, need to use `SystemRoute` (there's one other option, I don't know which one fits here)

Thank you in advance!

**Comment by Kern3l** [comment]
**An info if this project is still maintained would be greatly appreciated!** We like it very much, it helped us serve **a lot** more traffic than any sync solution, but the cap at _aiohttp==1.0.5_ became painful.

**Comment by ihavenonickname** [comment]
I think it's dead

let's move on

**Comment by klen** [comment]
The project is deprecated, yes.

BTW: It's updated to support aiohttp >= 3.4.4
